ci_scripts/install-holochain.sh

```diff
@@ -1,14 +1,14 @@
 #!/bin/bash
 
-REV=63b9fca72bf88b5760ad106256d3882f143d4f25
+REV=5812b41398e857134f780621dc1aa7bc6c05c5f0
 LAIR_REV=be5868e6dcbe99c795a101c0e27ba6ed5edd557d
 
-export CARGO_TARGET_DIR="$PWD/target"
-echo $CARGO_TARGET_DIR
+#export CARGO_TARGET_DIR="$PWD/target"
+#echo $CARGO_TARGET_DIR
```

Comment:
I made sure that all of the rust things used `target/` in the root of the repo for two reason:
1. Faster compilation since they can share any build artifacts from shared dependencies
2. Caching. [Currently our github actions caching setup only caches `target/` at the root](https://github.com/holochain/tryorama/blob/770b47cabc93770cec88fd6d138cf6c1e422f672/.github/workflows/test.yml#L33)

Is there a reason not to go back to all using the same target folder? If there is then please add the other target folders to the cached folders list. (Not sure what target folder `cargo install` uses by default, it's probably a global per-user folder)

Reply:
Ah, that makes sense.  I did this just to match what I was doing over in holochain-conductor-api, I'll go back to what you did.

Reply:
I guess I should make this the standard there too.